src/editor/HexEditor.tsx

```diff
@@ -20,11 +20,21 @@ const offsetSize = 8
 const bitsCountToNexLine = 16
 const arrayInitializer = 0
 const leftBytesPadding = '0'
+const splitInChunksOfTwo = /.{2}/g
 
 const getOffsetSizeFromBuffer = (data: Buffer): number =>
   Math.ceil(data.byteLength / offsetSize)
 
-const generateHexLines = (data: Buffer): string[] =>
+const chunkString = (text: string, chunkSize: number): string[] => {
```

Comment:
Maybe the name chucks will conflict with the chunks for reading the physical file, can we find a better name?

Reply:
Fixed!